### PR TITLE
fix/YPRO-106-screen-rotation-crashes-for-ViewAbility-and-Audibility

### PR DESCRIPTION
**Pull request by pabloserranof**

### CODE REVIEW

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java [hunk 1]
```diff
@@ -54,8 +55,8 @@ public AudioVolumeObserver(Context context) {
     }
 
     public void start(OnAudioVolumeChangedListener listener) {
-        stop();
         Handler handler = new Handler();
+        Context context = ((ViewGroup) listener).getContext();
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: Unused value

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java [hunk 2]
```diff
@@ -54,8 +55,8 @@ public AudioVolumeObserver(Context context) {
     }
 
     public void start(OnAudioVolumeChangedListener listener) {
-        stop();
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: This unregisters any previous registered listeners, which I think might be useful. Did this break something?

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java (hunk 1 above)
Comment (pabloserranof) [reply mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: ? context it is used in the next line

Comment (pabloserranof) [reply mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: @dzolnai ?

Comment (dzolnai) [reply mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: But it is also a private field. Remove one of the two.

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java [hunk 3]
```diff
@@ -55,8 +54,12 @@ public AudioVolumeObserver(Context context) {
     }
 
     public void start(OnAudioVolumeChangedListener listener) {
+        stop();
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/AudioVolumeObserver.java]: put this below context check, otherwise it will crash in stop

**Pull request merged**